packages/bierzo-wallet/src/components/wallet/Wallet.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import Continue from './Continue';
+import CurrencyToSend from './CurrencyToSend';
+import Note from './Note';
+import ReceiverAddress from './ReceiverAddress';
+
+const Wallet = () => {
+  const styleGrid: React.CSSProperties = {
+    backgroundColor: '#f9f9f9',
+    height: 'auto',
+    minHeight: '100vh',
+    display: 'grid',
+    gridTemplateColumns: '1fr minmax(375px, 450px) 1fr',
+    gridTemplateRows: 'repeat(4, auto)',
+    gridTemplateAreas: `
```

Comment:
I am curious to see this in action. Why are you following this approach? Can you update the PR and include an a couple of screenshots?

Reply:
Updated the PR with screenshots.